index.js

```diff
@@ -0,0 +1,106 @@
+// First, make sure to make all buttons for caclautor,
+// meaning numbers, getTotal(),getHistory(), clear, undo,operants
+// replace cuurent js code with previous js code
+// Make a current operant and previous operant attribute  on screen, both equal to ""
+// for each number click, concatnate the value string of that number to the current operant
+// maybe use .innerHtml or something reaserch what you can use
+// then when an operant is clicked, cuurent operasant is now previous operant and
+// then modfiy current operant to be "". also save the operant that was clicked on a varaible
+// then when numbers are clicked, it should be concatnated to currentOperant.
+// then when equalSButton  is clicked, we done a switch staement to check which operant is used and
+// use corresponding function on calcualtor to calculate previousopprant, currentOperant with the operant
+// then call .getTotal()
+
+const screen = document.querySelector("#screen");
+const numbers = document.querySelectorAll("[numbersToPress]");
+const operationButtons = document.querySelectorAll(".operationButtons");
+const equalsButton = document.querySelector("#equalsButton");
+const clearButton = document.querySelector("#clearButton");
+const historyButton = document.querySelector("#getHistory");
+const decimalButton = document.querySelector("#decimalButton");
+let secondValue = "";
+let operant = "";
+let currentValue = 0;
+let operantPressed = 0;
```

Comment:
should say operandPressed

Reply:
Also if your intention is for this to be a true/false value, then why do you have it set to 0 and why do you do -- on it? A bit strange really. I would normalise on either true or false, not mix it with numbers

Reply:
Good point, thanks for brining that up. Fixed it.